ansible/group_vars/all

```diff
@@ -16,6 +16,7 @@ runtimesManifest:
     default: true
   python:
   - kind: "python"
```

Comment:
For reference: https://github.com/openwhisk/openwhisk/blob/master/common/scala/src/main/scala/whisk/core/entity/Exec.scala#L167

Hence we need a test that confirms a kind `python` action will run as python:2 and otherwise `python:3` as python:3.

The manifest allows you to name the image - I'd have to look further to confirm but the the images that are prefixed with `openwhisk/` are recognized by the system and are rewritten to use an internal registry.